tools/ci/annotate_dm.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import re
+import os.path as path
+
+# Usage: python3 annotate_dm.py [filename]
+# If filename is not provided, stdin is checked instead
+
+raw_output = ""
+
+if len(sys.argv) > 1:
+    if not path.exists(sys.argv[1]):
+        print(f"Annotations file '{sys.argv[1]}' does not exist")
```

Comment:
For clarity, this should be colored so it stands out in comparison to the rest of the stuff that the terminal puts out. you can look at `ticked_file_enforcement` and its color "macro spans" for a better idea